src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -313,6 +313,35 @@ ParamArray& MainWindow::get_settings()
     return m_settings;
 }
 
+QDockWidget* MainWindow::create_dock_widget(const char* dock_name)
+{
+    QDockWidget* dock_widget = new QDockWidget(this);
+
+    const QString object_name = QString(dock_name).toLower().split(' ').join("_");
+    dock_widget->setObjectName(object_name);
+    dock_widget->setWindowTitle(dock_name);
+
+
+    const QList<QAction*>& actions = m_ui->menu_view->actions();
+    QAction* menu_separator = actions.last();
+    for (size_t i = (size_t) actions.size() - 2; i != 0; --i)
```

Comment:
Use `static_cast<>()`.
Add curly braces around single-statement but multi-line `for` loop body.